src/domain/enrolment/__tests__/CreateEnrolmentPage.test.tsx

```diff
@@ -263,11 +263,11 @@ test('renders form and user can fill it and submit', async () => {
   });
 
   await screen.findByRole('row', {
-    name: '25.09.2020 pe 12:30 – 12:30 Kirjasto 30 / 30',
+    name: '25.9.2020 pe 12:30 – 13:30 Kirjasto 30 / 30',
   });
 
   await screen.findByRole('row', {
-    name: '26.09.2020 la 13:20 – 12:30 Kirjasto 30 / 30',
+    name: '26.9.2020 la 13:20 – 14:20 Kirjasto 30 / 30',
```

Comment:
Mock data had bad end date for occurrence. Fixed so that by default end time is 1 hour after start time if is not given to the mock generator function.